src/app/app-providers/map-source-providers/open-layers-IDAHO-source-provider.ts

```diff
@@ -21,13 +21,28 @@ export class OpenLayerIDAHOSourceProvider extends BaseMapSourceProvider {
 	}
 
 	create(metaData: any): any {
+		/*
+		const source = layer.getSource();
+
+		 */
+		const source = new ol.source.XYZ({
+			url: metaData.imageUrl,
+			crossOrigin: 'Anonymous',
+			projection: 'EPSG:3857'
+		});
+
+		source.once('tileloadstart',() => {
```

Comment:
Please add comment for future uses. If it will affect the performance of the application.